src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -356,6 +356,17 @@ static inline void InvalidateOutputs(const std::vector<NDArray> &arrs,
   }
 }
 
+static inline std::vector<NDArray> CreateDefaultInputs(const std::vector<NDArray> &arrs) {
```

Comment:
Also add TODO to remove this unnecessary function when final solution is implemented.